pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -461,84 +462,64 @@ def sort_trigs(trial_dict, trigs, slide_dict, seg_dict):
     sorted_trigs = {}
 
     # Begin by sorting the triggers into each slide
-    # New seems pretty slow, so run it once and then use deepcopy
-    tmp_table = glsctables.New(glsctables.MultiInspiralTable)
     for slide_id in slide_dict:
-        sorted_trigs[slide_id] = copy.deepcopy(tmp_table)
-    for trig in trigs:
-        sorted_trigs[int(trig.time_slide_id)].append(trig)
+        sorted_trigs[slide_id] = []
+    for i, _ in enumerate(trigs['network/event_id']):
```

Comment:
Could this be instead
```python
for i in range(len(trigs['network/event_id'])):
```

Reply:
I am a bit puzzled by this code though. Imagine that `trigs['network/event_id']` contains [2,3,0,1]. This would mean that `i` would go through the list [0,1,2,3] instead. Is that the correct thing to do? Somewhat confusing that this code does not appear to actually use the values in `event_id`.

Reply:
I think working with direct indices would be faster rather than having to do something like `trigs['network/reweighted_snr']['network/event_id' == event_id]`. I'm thinking it would be more intuitive to have this function output the `event_id` though, so in this case I think I'll change that.

Reply:
Not sure I follow. Presumably the code using the actual `event_id`s would be as simple as
```python
for i in trigs['network/event_id']:
```
Is that not the case?

Reply:
Good point. The only time the index gets used is to pull the `slide_id`. This operation is likely faster, but probably not worth using `enumerate` for.

Reply:
Ah, I think I understand the goal now. Each trigger has an entry in the `event_id` and `slide_id` datasets (at the same index in the two) and you want to group the triggers by `slide_id`. So what you have now is probably very slow, but how about this:
```python
    for slide_id, event_id in zip(trigs['network/slide_id'], trigs['network/event_id']):
        sorted_trigs[slide_id].append(event_id)
```
Also, note that in a couple places you do `'network/event_id' == event_id` which does not make sense, you probably meant `trigs['network/event_id'] == event_id`.